CONTRIBUTING.md

```
@@ -225,7 +225,7 @@ If your issue does not clearly require one of these files, look for a narrower p
 
 ## Running Tests
 
-```bash
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)